qiskit/transpiler/preset_passmanagers/level1.py

```diff
@@ -307,7 +307,17 @@ def _opt_control(property_set):
             and pass_manager_config.layout_method is None
         ):
 
-            def _post_layout_condition(property_set):
+            def _run_post_layout_condition(property_set):
+                if _trivial_not_perfect(property_set):
+                    vf2_stop_reason = property_set["VF2Layout_stop_reason"]
+                    if (
+                        vf2_stop_reason is None
+                        or vf2_stop_reason != VF2LayoutStopReason.SOLUTION_FOUND
```

Comment:
Is there a reason to prefer comparing by equality here (`!=`) and identity (`is`) below on L326?

Reply:
Not really, besides being sloppy for enums either works: https://docs.python.org/3/library/enum.html#comparisons